src/modules/getBullsAndCows.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const getBullsAndCows = (userInput, randomNumber) => {
```

Comment:
The function assumes that the userInput and randomNumber are strings. Consider adding a validation to check if they are indeed strings and if not, convert them.

Reply:
I think there is no point in checking randomNumber. The function always returns string